Bug BZ1894835 Correct Defining Kibana index patterns (#28242)
openshift/openshift-docs: modules/cluster-logging-visualizer-indices.adoc

. Create your Kibana index patterns by clicking *Management* -> *Index Patterns* -> *Create index pattern*:

> ** Each user must manually create index patterns when logging into Kibana the first time in order to see logs for their projects. Users must create an index pattern named 'app' and use the `@timestamp` time field to view their container logs.

Reviewer: These should all be backticks instead of single quotes. Please update this line and the following one.

> ** Each user must manually create index patterns when logging into Kibana the first time in order to see logs for their projects. Users should create a new index pattern named *app* and use the `@timestamp` time field to view their container logs.

Reviewer: ```suggestion
** Each user must manually create index patterns when logging into Kibana the first time to see logs for their projects. Users must create an index pattern named `app` and use the `@timestamp` time field to view their container logs.
```

The only times we use bold styling are for UI elements and emphasis:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

> ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate.

> ** Each Admin user must create index patterns when logged into Kibana the first time for the *app*, *infra*, and *audit* indices using the `@timestamp` time field. 

Reviewer: ```suggestion
** Each admin user must create index patterns when logged into Kibana the first time for the `app`, `infra`, and `audit` indices using the `@timestamp` time field. 
```

Author: jbrigman completed further suggested modifications for BZ1894835: removing boldface inappropriately used.
